### DISCUSSION TITLE
Not able to import the CSS file

### DISCUSSION THREAD

**Discussion by ram2104** [remix-run/remix#3538] [Q&A] [answered]
Hey,

This is the weird issue I'm facing while importing the stylesheet in `root. tsx`. CSS is failing to load due to the object.
Ideally, the generated module should have a path to the generated CSS module. Not sure why is it mapping to an empty object rather build a file location.

<img width="512" alt="Screenshot 2022-06-22 at 9 04 19 PM" src="https://user-images.githubusercontent.com/19301579/175072571-71483d80-6f70-4bf7-8ca4-623e2bcfdd78.png">

**Comment by kiliman** [comment, marked as answer]
Remix will not compile your SCSS files automatically. You'll need to use `postcss` to do compile to a css file and import that. This is similar to how you would run tailwind compiler to generate the css file.
```ts
import stylesHref from '~/app/styles/styles.css'
export const links: LinksFunction = () => [
  { rel: 'stylesheet', href: stylesHref },
]
```

**Reply by ram2104** [reply]
It was an issue because I was using (override-esbuild-package) to override the ESbuild config for loading sass.

```
const { withEsbuildOverride } = require('remix-esbuild-override');
const { sassPlugin } = require('esbuild-sass-plugin');
```
But now I've switched to [CSS-preprocessor](https://remix.run/docs/en/v1/guides/styling#css-preprocessors). 
Thanks, @kiliman.
